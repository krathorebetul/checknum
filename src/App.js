import logo from './logo.svg';
import './App.css';
import CheckNum from './component/js/CheckNum';

function App() {
  return (
    <div className="App">
      <CheckNum/>
    </div>
  );
}

export default App;
