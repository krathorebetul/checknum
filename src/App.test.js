import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer';
describe("snapshot testing",()=>{
  test('renders learn react link', () => {
   
    const linkElement = renderer.create(<App/>).toJSON();
    expect(linkElement).toMatchSnapshot();
  });
}

)

