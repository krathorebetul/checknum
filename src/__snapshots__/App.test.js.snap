// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing renders learn react link 1`] = `
<div
  className="App"
>
  <div
    className="container mt-4"
  >
    <form
      className="container border border-danger col-sm-2 col-sm-offset-2"
      onSubmit={[Function]}
      style={
        Object {
          "backgroundColor": "#0B0B45",
        }
      }
    >
      <h4
        className="text-center text-warning"
      >
        Check Number Status
      </h4>
      <div
        className="container text-center mb-2"
      >
        <input
          className="form-control"
          name="num1"
          onChange={[Function]}
          placeholder="enter number"
          required={true}
          type="number"
        />
      </div>
      <div
        className="container text-center mb-2"
      >
        <input
          className="btn btn-primary"
          type="submit"
          value="checkNum"
        />
      </div>
      <div
        className="container text-center mb-2"
      >
        <input
          className="form-control"
          disabled={true}
          type="text"
          value=""
        />
      </div>
    </form>
  </div>
</div>
`;
